### PR TITLE
COM-365 Add key values in Visit Type dropdown in radiology upload and…

### PR DESCRIPTION
**Pull request by prashantsable1991**
Added key values in Visit Type dropdown in radiology upload and patient documents page.

### CODE REVIEW

File: ui/app/i18n/document-upload/locale_en.json [hunk 1]
```diff
@@ -17,5 +17,19 @@
     "YEARS_LABEL": "Years",
     "CHOOSE_ANSWER_FROM_DROPDOWN_LABEL": "Choose Answer",
     "DELETE_LABEL": "Delete",
-    "EDIT_LABEL": "Edit"
+    "EDIT_LABEL": "Edit",
+    "VISIT_TYPE_OPHTHALMOLOGY": "Ophthamology",
```
Comment (tmvumbi2) [inline ui/app/i18n/document-upload/locale_en.json]: @prashantsable1991 there is a small typo on Ophthalmology (missing "l" after 'Ophtha')

File: ui/app/i18n/document-upload/locale_fr.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{ 
+    "VISIT_TYPE_OPHTHALMOLOGY": "Ophtalmologie",
+    "VISIT_TYPE_LABORATORY": "Laboratoire",
+    "VISIT_TYPE_GYNECOLOGY": "Gynécologie",
+    "VISIT_TYPE_MATERNITY": "Maternite",
```
Comment (tmvumbi2) [inline ui/app/i18n/document-upload/locale_fr.json]: @prashantsable1991 small typo: missing accent on the last e: Maternité

File: ui/app/i18n/document-upload/locale_fr.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    "VISIT_TYPE_OPD": "Consultations Externes",
+    "VISIT_TYPE_CARDIOLOGY": "Cardiologie",
+    "VISIT_TYPE_ANC": "CPN",
+    "VISIT_TYPE_MINOR_SURGERY": "Petite Churigie",
```
Comment (tmvumbi2) [inline ui/app/i18n/document-upload/locale_fr.json]: @prashantsable1991 small typo, the last word should read: Chirurgie

File: ui/app/i18n/document-upload/locale_fr.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    "VISIT_TYPE_EMERGENCY": "Urgence",
+    "VISIT_TYPE_DELIVERY": "D'accouchement",
+    "VISIT_TYPE_DENTISTRY": "Dentisterie",
+    "VISIT_TYPE_RADIOLOGY": "Radiologie",
+    "VISIT_TYPE_PEDIATRIC": "Pediatrie",
```
Comment (tmvumbi2) [inline ui/app/i18n/document-upload/locale_fr.json]: @prashantsable1991 Missing accent: Pédiatrie

Comment (prashantsable1991) [reply ui/app/i18n/document-upload/locale_fr.json]: @tmvumbi2 added typos and missing accent. Please review once. Thanks.

**Pull request merged**